src/modules/addData.js

```diff
@@ -0,0 +1,26 @@
+const addScore = async (e) => {
+  e.preventDefault();
+  let valUser = document.querySelector('.user-name');
+  let valScore = document.querySelector('.user-score');
+  const itemUserVal = valUser.value;
+  const itemUserScore = valScore.value;
+  const apiData = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Al4d7IVkemOTTV/scores/';
+  try {
+    const finResponse = await fetch(apiData, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ user: itemUserVal, score: itemUserScore }),
+    });
+
+    valUser = '';
+    valScore = '';
```

Comment:
- [OPTIONAL] You did great trying to reset the inputs after submitting a score. :+1:  But currently, it seems not to be working and that's because you are setting the input element itself to `" "` and not the value. I'll suggest you update the code to `valUser.value = ''; and valScore.value = '';` and it will function just fine. :smile: :wave: